command/agent/winsvc/service_windows.go

```diff
@@ -0,0 +1,43 @@
+// +build windows
+
+package winsvc
+
+import (
+	wsvc "golang.org/x/sys/windows/svc"
+)
+
+type serviceWindows struct{}
+
+func init() {
+	interactive, err := wsvc.IsAnInteractiveSession()
```

Comment:
Looks like this func [is deprecated](https://godoc.org/golang.org/x/sys/windows/svc#IsAnInteractiveSession), and the recommendation is to use `IsWindowsService` instead. Found an example of the usage in [here](https://github.com/golang/sys/blob/master/windows/svc/example/main.go#L39).

Reply:
Ah, good catch, thanks!

Reply:
It turns out `IsWindowsService` hasn't been released yet. I updated this PR to use it and it won't compile, even with go 1.15.4. I'm going to revert that change.